tests/flytekit/unit/core/test_references.py

```diff
@@ -292,3 +293,11 @@ def inner_test(ref_mock):
     sdk_wf = get_serializable(OrderedDict(), serialization_settings, wf1)
     assert sdk_wf.nodes[1].workflow_node.launchplan_ref.project == "proj"
     assert sdk_wf.nodes[1].workflow_node.launchplan_ref.name == "app.other.flyte_entity"
+
+
+def test_lp_from_ref_wf():
+    @reference_workflow(project="project", domain="domain", name="name", version="version")
+    def ref_wf1(p1: str, p2: str) -> None:
+        ...
```

Comment:
should this codecov warning be ignored?

Reply:
I guess so, but its weird, what happens if you use `pass`